.github/workflows/container_test.yml

```diff
@@ -0,0 +1,25 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Docker container test
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+  pull_request:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  test:
+    name: Container test
+    runs-on: ubuntu-latest
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - uses: actions/checkout@v3
+      - name: Navigate to test script and run
+        run: cd .github/scripts && bash docker-create.sh -t
```

Comment:
i guess this needs some extra work; can we make sure that it installs java on the machine as a seperate step in the yml so we can build the app first and then create the container?